plugins/externalwrench/src/ApplyExternalWrench.cc

```diff
@@ -142,8 +153,9 @@ void RPCServerThread::run()
         m_rpcPort.read ( command,true );
         if ( command.get ( 0 ).asString() == "help" ) {
             this->m_reply.addVocab ( yarp::os::Vocab::encode ( "many" ) );
-            this->m_reply.addString ( "The defaul operation mode is with single wrench" );
+            this->m_reply.addString ( "The defaul operation mode is with single wrench without wrench smoothing" );
```

Comment:
`default`

If you could fix this legacy typo, it would be great. Thank you.

Reply:
Done https://github.com/Yeshasvitvs/gazebo-yarp-plugins/commit/b33cfcfca4a80c85ea300ba6cc3ec19a31ba8f8e